gateway/config/config.go

```diff
@@ -177,6 +179,10 @@ type EventingRule struct {
 	Options map[string]string `json:"options" yaml:"options"`
 }
 
+type SchemaObject struct {
+	Schema string `json:"schema" yaml:"schema"`
+}
+
```

Comment:
This should be in the handler

Reply:
Can not shift this since being used in Eventing struct in the same file